ports/atmel-samd/boards/matrixportal_m4/mpconfigboard.mk

```diff
@@ -13,6 +13,8 @@ LONGINT_IMPL = MPZ
 CIRCUITPY_LTO_PARTITION = one
 
 CIRCUITPY_AESIO = 0
+CIRCUITPY_FLOPPYIO = 0
+CIRCUITPY_GIFIO = 0
```

Comment:
Is this only gif writing? Seems like folks would want to read it on the matrix portal.

Reply:
Yes, gifio only writes GIFs.